bin/tsconfig.json

```diff
@@ -6,11 +6,13 @@
 		"target": "ES6",
 		"lib": [ "ES6", "ES2020.string" ],
 		"rootDir": ".",
-		"declarationMap": false,
 
-		// We're not interested in the output, but we must generate
-		// something as part of a composite project. Use the
-		// ignored `.cache` file to hide tsbuildinfo and d.ts files.
+		// These script are unpublished and packages do not depend on them.
```

Comment:
```suggestion
		// These scripts aren't published and packages don't depend on them.
```

Reply:
@sirreal Did you mean to commit @ZebulanStanphill's suggestion? Cause it seems to be _uncommitted_ :wink:

Reply:
Yes, I think I lost it in a rebase 🤦 

Thanks!